workshop.wwwapi/Models/Appointment.cs

```diff
@@ -2,13 +2,15 @@
 
 namespace workshop.wwwapi.Models
 {
-    //TODO: decorate class/columns accordingly
+    [Table("appointments")]
     public class Appointment
     {
-        
+        [Column("booking_date")]
         public DateTime Booking { get; set; }
         public int DoctorId { get; set; }
```

Comment:
you need doctor_id and patient_id columns on the database; make sure to specify the column names